GOVERNANCE.md

```diff
@@ -102,17 +149,21 @@ There is no strict minimum number of contributions needed to reach this level, a
 
 #### Nomination
 
-- To be nominated, a nominee is expected to already be performing some of the responsibilities of a Maintainer.
-- You can be nominated by any existing Maintainer (L2 or above). 
+- To be nominated, a nominee is expected to already be performing some of the duties and responsibilities of a Maintainer.
+- You can only be nominated by any existing Maintainer (L2 or above). 
 - Once nominated, there will be a vote by existing Maintainers. 
 - See [vote rules & requirements](#voting) for info on how the vote works.
 
 
 ### Level 3 (L3) - Core
 
-The **Core** role is available to community members who have a larger-than-usual impact on the Astro project and community. They are seen as leaders in the project and are listened to by the wider Astro community, often before they have even reached this level. A Core member is recognized for contributing a significant amount of time and energy to the project through issues, pull requests, bug fixes, implementing advanced enhancements/features, and/oractivly posting on Discord. 
+The **Core** role is available to community members who have a larger-than-usual impact on the Astro project and community. 
+
+They are seen as leaders in the project and are listened to by the wider Astro community, often before they have even reached this level. 
+
+A Core member is recognized for contributing a significant amount of their time and energy to the project through issues, pull requests, bug fixes, implementing advanced enhancements/features, and is activly posting on Discord. 
 
-Not every contributor will reach this level, and that's okay! L2 Maintainers still have significant responsibility and privileges in our community.
+Not every contributor will reach this level, and that's okay! L2 Maintainers still have significant responsibility and privileges as is within our community.
```

Comment:
```suggestion
Not every contributor will reach this level, and that's okay! L2 Maintainers still have significant responsibility and privileges within our community.
```

Reply:
As in where did this come from.... like I have no excuses for this utter brain fart💨